src/main/java/com/gbtec/Cell.java

```diff
@@ -0,0 +1,18 @@
+package com.gbtec;
+
+public class Cell {
+
+    private boolean isAlive;
+
+    public Cell() {
+        isAlive = false;
+    }
+
+    public boolean getStatus() {
```

Comment:
Setter and getter for a property should be named like the property itself or at least from the method name it should be clear which property is set. Here for example explanation how the methods should be named: https://stackoverflow.com/questions/5322648/for-a-boolean-field-what-is-the-naming-convention-for-its-getter-setter